ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceTaskViewModel.kt

```diff
@@ -78,12 +56,21 @@ class MultipleChoiceTaskViewModel @Inject constructor() : AbstractTaskViewModel(
     updateMultipleChoiceItems()
   }
 
-  fun toggleItem(item: MultipleChoiceItem) {
+  fun onOtherTextChanged(text: String) {
```

Comment:
Please add ktdoc to public functions and classes, same in all new non-test code please.

Reply:
Done